### ISSUE TITLE
Bug: OidcStrategy broken after #5013

### ISSUE THREAD

**Issue by shield-9** [weseek/growi#5091]
Environment
------------

### Host

| item     | version |
| ---      | --- |
|OS        |Debian GNU/Linux 9 (stretch) 9.13|
|GROWI     |4.5.8|
|node.js   |14.18.3|
|npm       |6.14.15|
|Using Docker|yes (K8s)|
|Using [growi-docker-compose][growi-docker-compose]|no|

[growi-docker-compose]: https://github.com/weseek/growi-docker-compose

*(Accessing https://{GROWI_HOST}/admin helps you to fill in above versions)*

### Client

| item     | version |
| ---      | --- |
|OS        |Windows 11 Version 21H2 (Build 22000.434)|
|browser   |Google Chrome	97.0.4692.71 (Official Build) （64 ビット） (cohort: Stable)|

How to reproduce? (再現手順)
---------------------------

1. Launch GROWI v4.5.4 (or lower)
1. Setup OIDC provider
    * Requirements for issuer host
        * Return valid content for `/.well-known/openid-configuration` with 200 status
        * **Return non-200 content for `/`**
1. Update to v4.5.5 or higher

What happens? (症状)
---------------------

- Signin attempts with OIDC provider fails since errored OidcStrategy setup

```
yarn run v1.22.15
$ node -r dotenv-flow/config node_modules/.bin/migrate-mongo up
Done in 0.80s.
    ...
{"name":"growi:service:PassportService","hostname":"growi-659f778fbd-jqkmz","pid":1,"level":20,"msg":"OidcStrategy: setting up..","time":"2022-01-15T10:56:02.300Z","v":0}
    ...
{"name":"growi:service:PassportService","hostname":"growi-659f778fbd-jqkmz","pid":1,"level":50,"msg":"OidcStrategy: issuer host unreachable: undefined","time":"2022-01-15T10:56:02.836Z","v":0}
{"name":"growi:service:PassportService","hostname":"growi-659f778fbd-jqkmz","pid":1,"level":50,"msg":"OidcStrategy: setup failed: OIDC Issur host unreachable","time":"2022-01-15T10:56:02.836Z","v":0}
```

What is the expected result? (期待される動作)
-------------------------------------------

- PassportService should not expect 200-status response from issuer host for `/`.
- PassportService should use OIDC's well-known URL for reachability check of issuer host.

Note
----

- 
-

**Comment by yuki-takei** [comment]
@shield-9 Thank you for your report.

We get the point and will fix ASAP in the next week. Please wait.

**Comment by yuki-takei** [comment]
Fixed by #5104
